modules/ob-creating-a-buildah-build.adoc

```diff
@@ -97,9 +97,9 @@ EOF
 .Example: Using `shp` CLI
 [source,terminal]
 ----
-$ shp build run buildah-golang-build --follow <1>
+$ shp build run buildah-golang-build --follow
```

Comment:
You had it right originally. You should use the code callout to describe the optional flag.